_drafts/rse-at-egu.md

```diff
@@ -0,0 +1,73 @@
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+The meeting kicked of with a short welcome by the initiator, Daniel, followed by [Jens Klump](http://people.csiro.au/Jens-Klump) (Science Leader Earth Science Informatics at CSIRO, Australia), Deputy President of EGU's [ESSI Division](http://essi.egu.eu/) and advisory board member of the Australian and New Zealand chapter [RSE-AUNZ](https://github.com/rse-aunz).
+Both pointed out the relevance of contributions to science made by software and subsequently the people contributing to that software in any way.
+
+The motivations for RSE national and international activities were further detailed by three representatives from national chapters. 
+David Topping from [UK RSE](http://rse.ac.uk/), the oldest and largest RSE organisation, took a look at the definition of an RSE, at the community history, and it's current state in the UK and beyond.
+Over 15 local groups exists and more are forming at a high rate, even leading to a competition for people.
+Martin Hammitzsch presented the German chapter, [de-RSE](http://de-rse.org/).
+Initiated only 1.5 years ago, he shared the group's objectives, how they work to build a community, challenges they face and some lessons learned: a great resource for the attendees from countries without any organisational structure yet.
+Third up was [Niels Drost](https://www.esciencecenter.nl/profile/dr.-niels-drost) (Netherlands eScience Center), who introduced the youngest European chapter [NL-RSE](http://nl-rse.org/) and its core team, which already generated a considerable reach across the Netherlands.
+
+<blockquote class="twitter-tweet" data-lang="de"><p lang="en" dir="ltr">4 national chapters, their history and plans, are introduced to ~40 participants, albeit meeting at the end of a long conference day. A good start for <a href="https://twitter.com/hashtag/EGU18RSE?src=hash&amp;ref_src=twsrc%5Etfw">#EGU18RSE</a> at <a href="https://twitter.com/hashtag/EGU18?src=hash&amp;ref_src=twsrc%5Etfw">#EGU18</a>. <a href="https://t.co/S2Ob54TSWe">pic.twitter.com/S2Ob54TSWe</a></p>&mdash; Daniel Nüst (@nordholmen) <a href="https://twitter.com/nordholmen/status/984484030031843328?ref_src=twsrc%5Etfw">12. April 2018</a></blockquote>
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+After the short talks, we took advantage of the group size, had a complete round of short introductions, and enganged in a discussion.
+We found a big challenge was reaching people engaged in RSEng activities at their work, especially outside of ESSI, and those for whom the "RSE" label does not ring a bell yet.
```

Comment:
found that one of the big challenges was to reach people

Reply:
"RSEng" first time here. Maybe explain via "(**R**esearch **S**oftware **E**ngineeri**ng**)"

Reply:
well spotted.